From 42982eb26f76050d0823efef8d39ad6d695d9234 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:04 -0400
Subject: [redhat] kabi: Adding symbol mempool_alloc_slab (mm/mempool.c)

Message-id: <505ea11a4971e5ef25a4c4b35536de9da7d3372a.1410510773.git.poros@redhat.com>
Patchwork-id: 95966
O-Subject: [PATCH RHEL7.1 124/233] kabi: Adding symbol mempool_alloc_slab (mm/mempool.c)
Bugzilla: 1062361 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1062361,1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Using kernels default slab allocator function passed to mempool_create.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/mempool_alloc_slab
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/mempool_alloc_slab
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/mempool_alloc_slab

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/mempool_alloc_slab b/redhat/kabi/kabi-module/kabi_ppc64/mempool_alloc_slab
new file mode 100644
index 0000000..1bf5253
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/mempool_alloc_slab
@@ -0,0 +1,2 @@
+#1-
+0x183fa88b mempool_alloc_slab vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/mempool_alloc_slab b/redhat/kabi/kabi-module/kabi_s390x/mempool_alloc_slab
new file mode 100644
index 0000000..1bf5253
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/mempool_alloc_slab
@@ -0,0 +1,2 @@
+#1-
+0x183fa88b mempool_alloc_slab vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/mempool_alloc_slab b/redhat/kabi/kabi-module/kabi_x86_64/mempool_alloc_slab
new file mode 100644
index 0000000..1bf5253
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/mempool_alloc_slab
@@ -0,0 +1,2 @@
+#1-
+0x183fa88b mempool_alloc_slab vmlinux EXPORT_SYMBOL
-- 
1.7.1